check.py

```diff
@@ -423,6 +424,7 @@ def fix(x):
 
 s2wasm.test_s2wasm()
 s2wasm.test_linker()
+# wasm2asm.test_wasm2asm()
```

Comment:
Yes, since there aren't many tests right now anyway. I was planning to enable the tests in a future PR that would finish 32 bit support and add more tests.